packages/sentry/src/sentry.spec.ts

```diff
@@ -7,6 +7,30 @@ const { testkit, sentryTransport } = sentryTestkit();
 
 const DUMMY_DSN = 'https://acacaeaccacacacabcaacdacdacadaca@sentry.io/000001';
 
+// At the moment, jsdom doesn't support PromiseRejectionEvent type. That leads to
+// ReferenceError in tests. So, we add this ad-hoc polyfill just for tests for now.
+// Once the following issue is fixed (which is unlikely to happen any soon) in jsdom
+// we can drop this polyfill: https://github.com/jsdom/jsdom/issues/2401
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+type Reason = any;
+type Payload = any;
```

Comment:
Better to use `unknown` than `any`. Then eslint won't complain.

```suggestion
type Reason = unknown;
type Payload = unknown;
```